NOTICE

```diff
@@ -18,18 +18,27 @@ This software includes third party software subject to the following licenses:
   Apache HttpCore under Apache License, Version 2.0
   Apache JempBox under Apache License, Version 2.0
   Apache PDFBox under Apache License, Version 2.0
+  Bean Validation API under The Apache Software License, Version 2.0
   Bouncy Castle PKIX, CMS, EAC, TSP, PKCS, OCSP, CMP, and CRMF APIs under Bouncy Castle Licence
   Bouncy Castle Provider under Bouncy Castle Licence
   Byte Buddy (without dependencies) under The Apache Software License, Version 2.0
   Byte Buddy Java agent under The Apache Software License, Version 2.0
+  ClassMate under The Apache Software License, Version 2.0
   Digipost API Client under The Apache Software License, Version 2.0
   Digipost Caching under The Apache Software License, Version 2.0
   Digipost Digg under The Apache Software License, Version 2.0
   Digipost HTTP Client Builder under The Apache Software License, Version 2.0
   Digipost Printability Validator under The Apache Software License, Version 2.0
+  digipost-data-types under The Apache Software License, Version 2.0
```

Comment:
Masse pen navngiving av våre open source-biblioteker. Og digipost-data-types. :trollface:

Reply:
🤣

Reply:
😆 digipost-data-types is for ro-bots not for hu-mans 🤖